docs/runbooks/research-reports-integration.md

```
@@ -223,3 +223,103 @@ uv run pytest tests/test_invest_api_feed_research_router_safety.py -v -m unit
 * Response field allowlist asserted in `test_invest_api_feed_research_router.py::test_response_field_allowlist`.
 * Copyright guardrail recursive scan in `test_invest_api_feed_research_copyright_guardrails.py::test_response_excludes_body_fields`.
 * Production smoke acceptance is gated on ROB-178's ingest smoke evidence.
+
+## ROB-207: Operator Scheduler / Bridge Activation
+
+### HTTP Bulk Ingest Endpoint
+
+The bridge endpoint accepts `POST /trading/api/research-reports/ingest/bulk` authenticated with a shared token:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)